### PR TITLE
Proof of donation for Nimuë Lady of the Lake Pool

### PR DESCRIPTION
**Pull request by brouwerQ**

### CODE REVIEW

File: NIMUE/donations.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  recipient: 
+    name: Dierenartsen Zonder Grenzen vzw (Vétérinaires Sans Frontières Belgium)
+    address: Kunstlaan 7-8, 1210 Brussel, Belgium
+    tel: +32 2 539 09 89
+    contact: info@vsf-belgium.org
+  ada: 91.50
+  usd: 118.85
+  timestamp: 1627601493
+  files:
+    - NIMUE/donation-6.png
```
Comment (kiriakos) [inline NIMUE/donations.yml:78]: I cannot find where this file proofs a transaction to `Dierenartsen Zonder Grenzen vzw (Vétérinaires Sans Frontières Belgium)`. I see that the date and values match but the receiver eludes me. Am I missing something?

Comment (brouwerQ) [reply NIMUE/donations.yml:78]: It says VSF, which is the abbrevation from Vétérinaires Sans Frontières. I don't know why this donation uses their English name and the previous one their Dutch name.

Comment (brouwerQ) [reply NIMUE/donations.yml:78]: It's what is on my credit card receipt. The organisation uses a provider to process payments, koalect.com, that's the meaning of the KO at the beginning. Then there's the name of the charity, VSF, and then a unique id for the tx and then what seems to be a telephone number form Koalect.

Comment (kiriakos) [reply NIMUE/donations.yml:78]: Fair enough.

**Review by kiriakos — APPROVED** [review]
LGTM

**Pull request merged**